src/Tribe/Settings_Helper.php

```diff
@@ -5,170 +5,165 @@
 	die( '-1' );
 }
 
+// Not sure if this is needed if we have the check below.
+// Without the check below this didn't seem to do anything.
 if ( class_exists( 'Tribe__Settings_Helper' ) ) {
```

Comment:
Not needed. Since the way classes are checked, you need to check for if it exists as you did below. Unless you put the check before the include.

Reply:
Done